DataAccess/Repository/TimeOffRequestRepository.cs

```diff
@@ -0,0 +1,54 @@
+﻿using DataAccess.Context;
+using DataAccess.Repository.Interfaces;
+using Domain.EF_Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccess.Repository
+{
+    public class TimeOffRequestRepository : IRepository<TimeOffRequest>
```

Comment:
why u removed base repository? Use generic _dbconextt.Set<> TEntity for any db model. U don't need copy paste same code to TimeOffRequestRepository, TimeOffRequestReviewRepository  and UserRepository